connaisseur/flask_server.py

```diff
@@ -59,12 +60,61 @@ def mutate():
     return jsonify(response)
 
 
+# health probe
 @APP.route("/health", methods=["GET", "POST"])
 def healthz():
     """
-    Handles the '/health' path and checks the health status of the flask and
-    the notary server. Sends back either '200' for a healthy status or '500'
+    Handles the '/health' endpoint and checks the health status of the flask
+    server. Sends back either '200' for a healthy status or '500'
     otherwise.
     """
-    health = health_check(os.environ.get("NOTARY_SERVER"))
-    return ("", 200) if health else ("", 500)
+    return ("", 200)
+
+
+# readiness probe
+@APP.route("/ready", methods=["GET", "POST"])
+def readyz():
+    """
+    Handles the '/ready' endpoint and checks the health status of the configured notary
+    server and whether the webhook is installed or not. If the notary is available and the
+    webhook is installed it returns a 200 status code. Otherwise should one of them not be
+    reachable, 500 is returned. For installation purposes, the readiness probe first
+    checks whether a specific bootstrap pod is running in the namespace. If this pod can
```

Comment:
Clarify that the "specific bootstrap pod" is what you're calling "sentinel" below